app/views/govuk_publishing_components/components/_layout_super_navigation_header.html.erb

```diff
@@ -78,6 +78,13 @@
         hidden
         id="super-navigation-menu-toggle"
         type="button"
+        data-ga4="<%= {
+          "event_name": "select_content",
+          "type": "super navigation header",
+          "text": "Menu",
+          "index": 1,
+          "index_total": navigation_links.length + 2
+        }.to_json %>"
```

Comment:
Would it be possible to add a `section` property here and set it to the `text` value? There's a comment in the implementation guide saying this would be useful.